docs/customosbuildinstructions.md

```diff
@@ -13,59 +13,59 @@ A LCOW custom Linux OS image was divided into two parts: a Linux kernel module a
 
 - Use the recommended [Kconfig](../kernel/kernel_config-4.11.x) to include all LCOW necessary kernel components
 
-- Build your kernel 
+- Build your kernel
 
 
     Note:  The key delta between the upsteam default setting and above kconfig is in the area of ACPI/NIFT/NVDIMM/OverlyFS/9pFS/Vsock/HyerpV settings, which were set to be built-in instead of modules.
-           The Kconfig above is still a work in process in terms of trimming any unnecessary components from the kernel image.  
+           The Kconfig above is still a work in process in terms of trimming any unnecessary components from the kernel image.
 
 ## How to construct user-mode components
 
 The expected user mode directory structure is required to constructed as follows: [See the complete user-mode file list](../kernel/completeUsermodeFileLists.md)
 
 Under the / directory, it should have the following subdirectories:
 
-- /tmp 
-- /proc 
-- /bin 
-- /dev 
-- /run 
-- /etc 
-- /usr 
-- /mnt 
-- /sys    
-
-- /init 
-- /root 
-- /sbin 
-- /lib64 
-- /lib      
+- /tmp
+- /proc
+- /bin
+- /dev
+- /run
+- /etc
+- /usr
+- /mnt
+- /sys
+
+- /init
+- /root
+- /sbin
+- /lib64
+- /lib
 
 Here are the expected contents of each subdirectory /file
-     
-1. Subdirectories with **empty** contents:  /tmp /proc /dev /run /etc /usr /mnt /sys 
 
-2. **/init** 
+1. Subdirectories with **empty** contents:  /tmp /proc /dev /run /etc /usr /mnt /sys
+
+2. **/init**
    This is the [init script file](../kernel/scripts/init_script)
 
-3. **/root** : this is the home directory of the root account. 
+3. **/root** : this is the home directory of the root account.
 
-4. **/sbin** : 
-    - /sbin/runc  
+4. **/sbin** :
+    - /sbin/runc
 
-              Note:this is the "runc" binary for hosting the container execution environment. 
+              Note:this is the "runc" binary for hosting the container execution environment.
               It needs to be the following release
-              runc version 1.0.0-rc3
-              commit: 992a5be178a62e026f4069f443c6164912adbf09
-              spec: 1.0.0-rc5
+              runc version 1.0.0-rc4
+              commit: 3f2f8b84a77f73d38244dd690525642a72156c64
+              spec: 1.0.0
```

Comment:
The official commit for runc v1.0.0-rc4 is https://github.com/opencontainers/runc/commit/2e7cfe036e2c6dc51ccca6eb7fa3ee6b63976dcd
It might be a good idea to use it instead of 3f3f... or the one I added in setup_test_env ( 882d8eaba6682179e397c78d5a40e45fa4d571ce)

Could you syn them up?

Reply:
RunC is a separate binary that we pulled into linux initrd. This binary is not part of the opengcs repo.
So, no need to change any vendor file.

Reply:
Do you know what the advantage of using the rc4 commit rather than the commit docker uses would be?

Reply:
The official RunC release one should be more stable.

Reply:
Talked offline, going to use the version docker uses at least for now, since all of our testing, etc. is on docker master.